src/components/TeamMember/TeamMember.styles.ts

```diff
@@ -19,14 +19,26 @@ export const ExpandIcon = styled(Icon)`
 export const AvatarFrame = styled.div`
   overflow: hidden;
   width: 100%;
+  box-shadow: #cdcdcd 0px -1.5px 2px 0px;
+  background-color: white;
+`;
+
+export const AvatarCircle = styled.a`
+  display: flex;
+  width: 14rem;
+  height: 14rem;
+  border-radius: 50%;
+  overflow: hidden;
+  background-color: white;
+  margin: 1rem auto;
```

Comment:
the cursor pointer should only be in the avatar circle, and not in the entire box
```suggestion
  margin: 1rem auto;
  cursor: pointer;
```